server/storeAppointment.php

```diff
@@ -0,0 +1,120 @@
+<?
+require_once 'config.php';
+
+storeAppointment($_POST);
+
+function storeAppointment($data){
+	GLOBAL $DB_CONN;
+	$conn = $DB_CONN;
```

Comment:
btw what is the reasoning for creating a new variable to hold the connection, when DB_CONN already has it?